install_config/install/docker_registry.adoc

```diff
@@ -169,9 +169,41 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+Because the integrated registry is referenced by its service IP address,
+if you decide to delete and recreate the docker-registry service,
+you should either take care to re-use its address,
+or reboot the cluster after creating the new docker-registry service.
+
+[[re-using-the-address]]
+==== Re-using the Address
+
+To re-use the IP address, you must save the IP address of the old docker-registry
+service prior to deleting it, and arrange to replace the newly assigned IP address
+with the saved one in the new docker-registry service.
+
+// NB: Snarfed from <https://github.com/openshift/openshift-docs/issues/1494>.
+. Make a note of the `ClusterIP` for the service.
+. `oc delete svc/docker-registry dc/docker-registry`
+. `oadm registry ... -o yaml > registry.yaml`
+. Edit *_registry.yaml_*, find the `Service` there,
+and set its `ClusterIP` to the address noted in step 1.
+. `oc create -f registry.yaml`
 
+[[rebooting-the-cluster]]
+==== Rebooting the Cluster
+
+If you are unable to re-use the IP address,
+you must reboot the cluster after creating the new docker-registry service.
+In some circumstances it is sufficient to reboot only the master,
+but that may leave stale references to the old IP address in
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#master[etcd].
+
+
+[[viewing-logs]]
 == Viewing Logs
 
 To view the logs for the Docker registry, run the `oc logs` indicating the desired pod:
```

Comment:
I suppose even better would be to omit the inline markup.  But, as you note, this is out of scope for this PR.  @vikram-redhat WDYT?